server/src/main/java/org/candlepin/pki/impl/BouncyCastlePKIReader.java

```diff
@@ -190,9 +182,6 @@ else if (caKeyObj instanceof PrivateKey) {
                         caKeyObj.getClass().getName());
                 }
             }
-            finally {
-                reader.close();
```

Comment:
Don't we need to close the reader somewhere?

Reply:
This switched to the "new" try-with-resources block, which automatically handles any closeables within the try declaration on line 165.